### DISCUSSION TITLE
크롬에 하면 안되나요 웨일에다가 해야하나요

### DISCUSSION THREAD

**Discussion by 38sun** [changdoc/naver-vaccine-macro#75] [Q&A]
크롬 에다가 하니 작동은 되는것 같은데요
웨일에 해야지만 인증 오류같은것이 아나고
잘 작동이 되는건지요

**Comment by OneTop4458** [comment]
크롬 웨일 둘다 지원 가능합니다.
크로미움 기반 브라우저면 다 실행 가능합니다.

**Reply by 38sun** [reply]
감사합니다
